src/main/java/edu/kis/powp/jobs2d/command/analyzer/ICommandUsageAnalyzer.java

```diff
@@ -0,0 +1,28 @@
+package edu.kis.powp.jobs2d.command.analyzer;
+
+import edu.kis.powp.jobs2d.command.DriverCommandVisitor;
+import edu.kis.powp.jobs2d.command.ICompoundCommand;
+
+/**
+ * This interface was defined for complex commands benchmarking and analysis of usage purpose.
+ */
+public interface ICommandUsageAnalyzer extends DriverCommandVisitor {
```

Comment:
Verify if DriverCommandVisitor needs to be a part of this interface

Reply:
updated in latest hotfix